From 229d24a95f8d2c933277b8e801a9578cfb6d3b6a Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Mon, 23 Jan 2017 12:26:32 +0200
Subject: [PATCH 1317/2241] arm64: defconfig: enable dm-crypt

dm-crypt is a disk encryption system using the kernels crypto API framework
and device mapper subsystem.
With dm-crypt, administrators can encrypt entire disks, logical volumes,
partitions, but also single files.

Change-Id: Iefedcaf84e73fc961dd17bccd3991658ae0eef5e
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35928
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35999
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index f54a549..2dd54f4 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -109,6 +109,8 @@ CONFIG_MD_RAID0=y
 CONFIG_MD_RAID1=y
 CONFIG_MD_RAID10=y
 CONFIG_MD_RAID456=y
+CONFIG_BLK_DEV_DM=y
+CONFIG_DM_CRYPT=y
 CONFIG_NETDEVICES=y
 CONFIG_BONDING=y
 CONFIG_MACVLAN=y
-- 
2.7.4

